### PR TITLE
Add VPC peering between the CyHy and BOD VPCs

### PR DESCRIPTION
**Pull request by jsf9k**

### CODE REVIEW

**Review by felddy — APPROVED** [review]

File: ansible/roles/mongo/tasks/main.yml [hunk 1]
```diff
@@ -28,6 +28,13 @@
     state: present
     line: After=network.target cloud-config.service
 
+- name: Configure mongo service to listen on all interfaces
+  lineinfile:
+    dest: /lib/systemd/system/mongod.service
+    regexp: '^( *bindIp=.*)$'
```
Comment (jsf9k) [inline ansible/roles/mongo/tasks/main.yml]: This isn't working.  :angry:   I need to fix this before we merge.

**Review by jsf9k — COMMENTED** [review]
Needs a little work before merging :disappointed:

**Comment by jsf9k** [comment]
Building up and tearing down is complicated now that we have two Terraform projects (`terraform` and `terraform _bod`) that are interdependent.  Maybe we should move the `terraform_bod` code into the `terraform` directory.

What do you guys think?

File: ansible/roles/mongo/tasks/main.yml (hunk 1 above)
Comment (dav3r) [reply ansible/roles/mongo/tasks/main.yml]: There are a couple other changes that I made to the mongo config file (located at /etc/mongod.conf, not /lib/systemd/system/mongod.service), so let's coordinate on those too.

**Comment by dav3r** [comment]
@jsf9k , I agree that if the `terraform` and `terraform_bod` directories are now interdependent, they should be merged.

**Comment by jsf9k** [comment]
@dav3r, I moved the `terraform_bod` code into the `terraform` directory (and renamed and modified some files as necessary).  Let's get those Mongo configuration changes you have in so we can merge this one!
:penguin: :tada: :trumpet: :smiling_imp: :cat2: :koala: :dolphin:

**Review by dav3r — APPROVED** [review]
👍 @jsf9k

**Pull request merged**